packages/expression-parser/src/expression-parser/ExpressionParser.js

```diff
@@ -205,4 +248,8 @@ export class ExpressionParser {
   getFunctionOverrides() {
     return {};
   }
+
+  factory(...args) {
+    return factory(...args);
```

Comment:
```suggestion
    return this.math.factory(...args);
```
Thoughts on this? It will allow us to use the factory function from our existing math instance, which means if we've already imported stuff into that math instance we can use it (custom types for example)

Reply:
Good call!